client/app/services/offline-listener.js

```diff
@@ -1,4 +1,6 @@
-function OfflineListener(toastr) {
+export let OfflineListener = null; // eslint-disable-line import/no-mutable-exports
```

Comment:
No need to export this one. Nothing uses it, it just registers itself to some event and then shows a notification when the user is offline.